lib/portlayer/storage/image_cache.go

```diff
@@ -237,6 +237,19 @@ func (c *NameLookupCache) Export(op trace.Operation, store *url.URL, id, ancesto
 	return c.DataStore.Export(op, store, id, ancestor, spec, data)
 }
 
+func (c *NameLookupCache) Import(op trace.Operation, store *url.URL, diskID string, spec *archive.FilterSpec, tarStream io.ReadCloser) error {
+	_, err := util.ImageStoreName(store)
+	if err != nil {
+		return err
+	}
+
+	if archiver, ok := c.DataStore.(archive.Archiver); ok {
```

Comment:
This pattern is no longer necessary, see https://github.com/vmware/vic/blob/master/lib/portlayer/storage/image_cache.go#L236